GitCommands/AppTitleGenerator.cs

```diff
@@ -46,10 +50,17 @@ public string Generate(string workingDir = null, bool isValidWorkingDir = false,
             var description = _description.Get(workingDir);
 
 #if DEBUG
-            return $"{description} ({branchName}) - Git Extensions [DEBUG]";
+            return $"{description} ({branchName}) - Git Extensions [{BuildBranch ?? "DEBUG"}]";
```

Comment:
I am afraid this will not compile.

Reply:
This works - if you request changes is another thing

Reply:
Perfect if it works. I had problems with similar constructs before.